communication/state_telemetry.cpp

```diff
@@ -203,6 +213,44 @@ static mav_result_t state_telemetry_set_mode_from_cmd(State* state, mavlink_comm
     return result;
 }
 
+static mav_result_t state_telemetry_set_arm_from_cmd(State* state, mavlink_command_long_t* packet)
+{
+    mav_result_t result = MAV_RESULT_ACCEPTED;
+
+    mav_mode_t new_mode;
+    float arm_cmd = packet->param1;
+
+    if (state->mav_state == MAV_STATE_CALIBRATING)
+    {
+        result = MAV_RESULT_TEMPORARILY_REJECTED;
+        print_util_dbg_print("Currently calibrating, impossible to change mode.\r\n");
+    }
+    else
+    {
```

Comment:
@ndousse Should we allow arm / disarm only when CTRL_MODE_SRC is set to ground station?
- option 1 (this PR): active all the time.
    * safety issue if we arm by mistake?
- option 2: when CTRL_MODE_SRC is not ground station. Allow only disarm from ground station
    * no arming by mistake, but disarm is still possible as backup 
- option 3: disable when CTRL_MODE_SRC is not ground station

I tend towards option 2.